source/configuration/read.txt

```diff
@@ -120,7 +120,13 @@ You can configure the following properties to read from MongoDB:
        |
        | **Default:** ``true``
 
-
+   * - ``outputExtendedJson``
+     - | When ``true``, the Connector converts BSON types not supported by Spark into 
```

Comment:
[suggestion] I think this should be lower case to match the way "connector" is used on the rest of the page. It is only capitalized when we say "MongoDB Spark Connector".